packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -57,7 +60,9 @@ function genConfig(compatVersion: number | string): CompatConfig[] {
 	};
 
 	const compatVersionStr =
-		typeof compatVersion === "string" ? compatVersion : `N${compatVersion}`;
+		typeof compatVersion === "string"
+			? `${compatVersion} (N)`
+			: `${getRequestedRange(baseVersion, compatVersion)} (N${compatVersion})`;
```

Comment:
You probably want the `testBaseVersion` here. I know we've had the wrong testing before because test branches are in `0.0.0-<buildId>-test`, so comparisons might be wrong.

Reply:
Also when I just did loader compat, I just added a separate function, we probably should do that as well.

Reply:
Won't this affect all the describeFullCompat tests?